pages/common/mosh.md

```diff
@@ -0,0 +1,33 @@
+# mosh
+
+> Mobile Shell (`mosh`) is remote terminal application that allows roaming.
+> `mosh` is a more robust and responsive replacement for SSH.
+> Visit <https://mosh.org> for more information.
+
+- Typical usage:
```

Comment:
Ah, oops! I confused this comment with the one about removing the link: https://github.com/tldr-pages/tldr/pull/1384/files/ff9e94078b1ade9daa585b3a4722e4879aebc3aa#diff-32fbf11af0de15b9a5933f4ffe23e376

Sorry! I will make the change.